Dockerfile-acceptance

```diff
@@ -7,6 +7,6 @@ COPY codecept.conf.js codecept.conf.js
 COPY apps apps/
 COPY test.sh test.sh
 
-RUN npm --loglevel warn install --ignore-scripts
+RUN npm --loglevel warn install
 
```

Comment:
because reasons. Reasons which I don't understand entirely, but reasons which stop the Bad Argument error happening on Drone